src/CTA.Rules.ProjectFile/ProjectFileCreator.cs

```diff
@@ -99,6 +100,27 @@ public class ProjectFileCreator
             }
         }
 
+        public ProjectFileCreator(string projectFile, List<string> targetVersions, Dictionary<string, string> packages,
```

Comment:
Might be cleaner to chain constructors here since everything past `_sourceVersions = sourceVersions;` looks to be the same as the content for the constructor above this one, but not a big deal either way.

Reply:
ouh good catch, fixed it. Thanks :-)